docs/core/porting/upgrade-assistant-uwp-framework.md

```diff
@@ -8,7 +8,7 @@ ms.date: 09/20/2022
 # Migrate UWP apps to Windows App SDK with the .NET Upgrade Assistant
```

Comment:
Actually, we do have support for UWP in the new VSIX and new CLI so we need to update the docs to suggest that. We can do it as a separate PR.